### PR TITLE
fix: File tree icons

### PR DESCRIPTION
**Pull request by orronai**
- Changed icons style

### CODE REVIEW

File: lms/models/solutions.py [hunk 1]
```diff
@@ -195,7 +195,7 @@ def get_files_tree(files: Iterable[SolutionFile]) -> List[Dict[str, Any]]:
             'id': file.id,  # type: ignore
             'fullpath': file.path,
             'path': os.path.split(file.path.strip('/'))[1],  # type: ignore
-            'indent': file.path.strip('/').count('/'),  # type: ignore
+            'indent': file.path.strip('/').count('/') + 1,  # type: ignore
```
Comment (yammesicka) [inline lms/models/solutions.py:198]: Why?

File: lms/static/file-icon-vivid.min.css [hunk 2]
```diff
@@ -0,0 +1,1118 @@
+/*!
```
Comment (yammesicka) [inline lms/static/file-icon-vivid.min.css:1]: Thtat's called `min` but it doesn't look like a minified version

**Review by yammesicka — CHANGES_REQUESTED** [review]

File: lms/models/solutions.py (hunk 1 above)
Comment (orronai) [reply lms/models/solutions.py:198]: No it doesn't. If we wouldn't do it there will be a collision with the folders/files icons in the first hierarchy with the comments number.
Another option is to add `1` to the count in the html file.

File: lms/static/file-icon-vivid.min.css (hunk 2 above)
Comment (orronai) [reply lms/static/file-icon-vivid.min.css:1]: As you can see in the documentation I downloaded it from here: https://fileicons.org/
I can change the file name if you want to `file_icons.css` or something like that.

**Review by yammesicka — APPROVED** [review]

**Pull request merged**